### PR TITLE
Hide active icon port from RU

### PR DESCRIPTION
**Pull request by Blundir**
## About The Pull Request

Makes hide action icon have purple border if it is active

dreamseeker_yz00w4M4m1

100% port of https://github.com/Cosmic-Overlord/TerraGov-Marine-Corps/pull/121 , which is inspired by the same thing on CM

## Why It's Good For The Game

More visuals = good

## Changelog
:cl:
add: xenomorph active hide action icon has a purple border now
/:cl:

### CODE REVIEW

**Comment by ivanmixo** [comment]
Why not just green like the other abilities use when they're active?

**Comment by Blundir** [comment]
> Why not just green like the other abilities use when they're active?

it's a lot more easier to differentiate, more comfortable in general, mess of the same color borders is confusing

**Comment by ivanmixo** [comment]
Seems a tad inconsistent

**Comment by Blundir** [comment]
> Seems a tad inconsistent

Not really, having multiple green highlighted abilities attracts too much attention, having multiple different colored ones makes it easy to identify at a glance

**Review by Kuro020 — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm [hunk 1]
```diff
@@ -143,6 +143,8 @@
 /datum/action/xeno_action/activable/pounce/proc/prepare_to_pounce()
 	if(owner.layer == XENO_HIDING_LAYER) //Xeno is currently hiding, unhide him
 		owner.layer = MOB_LAYER
+		for(var/datum/action/xeno_action/xenohide)
```
Comment (ivanmixo) [inline code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm:146]: Uh what the fuck? How is this a valid for loop? This loops thru the whole WORLD STOP IT

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm (hunk 1 above)
Comment (Blundir) [reply code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm:146]: no warning by the way

Comment (Blundir) [reply code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm]: oh wait i'm dumb, why it even compiles

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**